File: official/docs/golang/v3/api-keys/retrieve.go
package example

import (
	"fmt"

	"github.com/EasyPost/easypost-go/v3"
)

func main() {
	client := easypost.New("EASYPOST_API_KEY")

	// Retrieve all API keys including children
	apiKeys, _ := client.GetAPIKeys()

	fmt.Println(apiKeys)

	// Retrieve API keys for a specific child user
	childApiKeys, _ := client.GetAPIKeysForUser("user_...")

	fmt.Println(childApiKeys)
}

File: official/docs/csharp/current/api-keys/retrieve.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using Newtonsoft.Json;
using EasyPost;
using EasyPost.Models.API;

namespace EasyPostExamples
{
    public class Examples
    {
        public static async Task Main()
        {
            var client = new EasyPost.Client("EASYPOST_API_KEY");

            // Retrieve all API keys including children
            List<ApiKey> apiKeys = await client.ApiKey.All();

            Console.WriteLine(JsonConvert.SerializeObject(apiKeys, Formatting.Indented));

            // Retrieve API keys for a specific child user
            List<ApiKey> childApiKeys = await client.ApiKey.RetrieveApiKeysForUser("user_...");

            Console.WriteLine(JsonConvert.SerializeObject(childApiKeys, Formatting.Indented));
        }
    }
}
